docs/customization/overview.mdx

```diff
@@ -138,114 +138,19 @@ Remove the `cl-` prefix from a class and use it as the key for a new object in t
-
-      ```tsx {{ filename: 'app.tsx', mark: [[7, 13]] }}
-      import { ClerkProvider, SignIn } from '@clerk/nextjs'
-      import type { AppProps } from 'next/app'
-
-      function MyApp({ pageProps }: AppProps) {
-        return (
-          <ClerkProvider {...pageProps}>
-            <SignIn
-              appearance={{
-                elements: {
-                  formButtonPrimary: 'your-org-button org-red-button',
-                },
-              }}
-            />
-          </ClerkProvider>
-        )
-      }
-
-      export default MyApp
-      ```
-    </CodeBlockTabs>
-  </Tab>
-</Tabs>
+```tsx {{ mark: [4] }}
+<SignIn
+  appearance={{
+    elements: {
+      formButtonPrimary: 'your-org-button org-red-button',
+    },
+  }}
+/>
+```
 
 #### Use Tailwind classes to style Clerk components
 
-You can style the elements of a Clerk component with Tailwind.
-
-The following example shows how to style the primary button in a `<SignIn />` component with Tailwind classes:
-
-<Tabs items={["Next.js"]}>
-  <Tab>
-    <CodeBlockTabs options={["App Router", "Pages Router"]}>
-      ```tsx {{ filename: 'app/layout.tsx', mark: [[8, 14]] }}
-      import { ClerkProvider, SignIn } from '@clerk/nextjs'
-
-      export default function RootLayout({ children }: { children: React.ReactNode }) {
-        return (
-          <ClerkProvider>
-            <html lang="en">
-              <body>
-                <SignIn
-                  appearance={{
-                    elements: {
-                      formButtonPrimary: 'bg-slate-500 hover:bg-slate-400 text-sm',
-                    },
-                  }}
-                />
-              </body>
-            </html>
-          </ClerkProvider>
-        )
-      }
-      ```
-
-      ```tsx {{ filename: 'app.tsx', mark: [[7, 13]] }}
-      import { ClerkProvider, SignIn } from '@clerk/nextjs'
-      import type { AppProps } from 'next/app'
-
-      function MyApp({ pageProps }: AppProps) {
-        return (
-          <ClerkProvider {...pageProps}>
-            <SignIn
-              appearance={{
-                elements: {
-                  formButtonPrimary: 'bg-slate-500 hover:bg-slate-400 text-sm',
-                },
-              }}
-            />
-          </ClerkProvider>
-        )
-      }
-
-      export default MyApp
-      ```
-    </CodeBlockTabs>
-  </Tab>
-</Tabs>
-
-> [!NOTE]
-> To use Tailwind CSS v4, you need to make use of the `cssLayerName` property to ensure that Tailwind's utility styles are applied after Clerk's styles.
-
-The following example shows how to use the `cssLayerName` property to ensure that Tailwind's utility styles are applied after Clerk's styles:
+To use Tailwind CSS v4, you must set the `cssLayerName` property to `clerk` to ensure that Tailwind's utility styles are applied after Clerk's styles. It's recommended to add this to the `<ClerkProvider>` that wraps your app so that it's applied to all Clerk components, as shown in the following example:
```

Comment:
It's not required that its set to `clerk`. They can name the layer whatever they want via the `cssLayerName` property.